interop-testing/src/test/java/io/grpc/testing/integration/ProxyTest.java

```diff
@@ -61,7 +61,7 @@ public void shutdownTest() throws IOException {
 
   @Test
   public void smallLatency()
-      throws UnknownHostException, IOException, InterruptedException, ExecutionException {
+      throws IOException, InterruptedException, ExecutionException {
```

Comment:
In contrast to APIs, for test methods we can simply declare `throws Exception`, because any exception being thrown will immediately fail the test and we can know what went wrong from the test result.